types/jest/index.d.ts

```diff
@@ -169,7 +170,7 @@ declare namespace jest {
     /**
      * Creates a mock function similar to jest.fn but also tracks calls to object[methodName]
      */
-    function spyOn<T extends {}, M extends keyof T>(object: T, method: M): SpyInstance<T[M]>;
+    function spyOn<T extends {}, M extends keyof T>(object: T, method: M, accessType?: string): SpyInstance<T[M]>;
```

Comment:
`accessType?: 'get' | 'set'` would be more accurate here.

Reply:
corrected :)

Reply:
Even though it makes for a property accessor to be either `get` or `set`, the `spyOn` itself doesn't expect `accessType` to be accurate:

https://github.com/facebook/jest/blob/490f435d0e2a5478f3817899f66ee39817a1a18d/packages/jest-mock/src/index.js#L725-L729

In fact, the `spyOn` signature expects any `string` for `accessType`:

https://github.com/facebook/jest/blob/490f435d0e2a5478f3817899f66ee39817a1a18d/packages/jest-mock/src/index.js#L662